released_dataset/README.md

```diff
@@ -121,6 +121,32 @@
 | MMSA EPF-WDS interdependency network - NCI | 631f95f3b4f2e17eff72e2e9 | September 14, 2022 | networkInterdependencyTable | |
 | MMSA WDS-EPF interdependency network - NCI| 631f4bfb923e0e341acf1c96 | September 14, 2022 | networkInterdependencyTable | |
 
+## SLC
+### Building inventory 
+| Name | Dataset ID | Date Created | Data Type                  | Note |
+| ------------------------------------ | ------------------------ |--------------|----------------------------| ---- |
+| Building Inventory| 62fea288f5438e1f8c515ef8 | Feb 22, 2023 | ergo:buildingInventoryVer6 | |
+
+### Other datasets
+| Name                                | Dataset ID | Date Created | Data Type                 | Note |
+|-------------------------------------| ------------------------ |--------------|---------------------------| ------------ |
+| Salt Lake County Liquefaction Susceptibility | 62fe9ab685ac6b569e372429 | Feb 22, 2023 | ergo:liquefactionSusceptibilityPolygon | CGE analysis |
```

Comment:
Not all of these are for the CGE analysis. I think most could generically be said for Salt Lake City Community App maybe?

Reply:
> Not all of these are for the CGE analysis. I think most could generically be said for Salt Lake City Community App maybe?

I tried to update but I am not 100 percent sure if I did it correctly. Please review if those are correct